_includes/v23.1/cdc/privilege-model.md

```diff
@@ -11,9 +11,23 @@ You can [grant](grant.html#grant-privileges-on-specific-tables-in-a-database) a
 GRANT CHANGEFEED ON TABLE example_table TO user;
 ~~~
 
-This privilege model provides a more granular way to grant users the ability to create a changefeed on a table. A user granted the `CHANGEFEED` privilege can create changefeeds on the target table even if the user does **not** have the [`CONTROLCHANGEFEED` role option](alter-role.html#role-options) or the `SELECT` privilege on the table. 
+When you grant a user the `CHANGEFEED` privilege, they can:
 
-Since you can grant the `CHANGEFEED` privilege to a user or role **without** them needing the `SELECT` privilege on a table, these users will be able to create changefeeds, but they will not be able to run a `SELECT` query on that data directly. {% if page.name == "create-changefeed.md" %} However, these users could still read this data indirectly if they have read access to the [sink](changefeed-sinks.html), or create a "sinkless" changefeed that emits messages to the SQL session.{% else %} However, these users will be able to read this data indirectly as `EXPERIMENTAL CHANGEFEED FOR` emits changefeed messages to the SQL session. {% endif %}
+- Create changefeeds on the target table even if the user does **not** have the [`CONTROLCHANGEFEED` role option](alter-role.html#role-options) or the `SELECT` privilege on the table. 
+- {% include_cached new-in.html version="v23.1" %} Manage the changefeed jobs that they have created using the [`SHOW CHANGEFEED JOB`](show-jobs.html), [`PAUSE JOB`](pause-job.html), [`RESUME JOB`](resume-job.html), and [`CANCEL JOB`](cancel-job.html) commands.
+
+These users will be able to create changefeeds, but they will not be able to run a `SELECT` query on that data directly. However, they could still read this data indirectly if they have read access to the [sink](changefeed-sinks.html).
+
+{% include {{ page.version.version }}/cdc/ext-conn-cluster-setting.md %}
+
+### Privilege model
+
+Granted privileges | Usage 
+-------------------+-------
+`CHANGEFEED` | Create changefeeds on tables defined in the `GRANT` statement.<br>Manage the changefeed job.
```

Comment:
`CHANGEFEED` on tables | Create changefeeds on tables. Manage changefeed jobs for these tables.
`CHANGEFEED` on tables + `USAGE` on external connections | ...
`SELECT` on tables | ...
**Deprecated** `CONTROLCHANGEFEED` role option + `SELECT` on tables | Create changefeeds on tables.